onnxruntime/core/providers/cuda/cuda_execution_provider.cc

```diff
@@ -1169,7 +1171,7 @@ static void RegisterCudaKernels(KernelRegistry& kernel_registry) {
   for (auto& function_table_entry : function_table) {
     kernel_registry.Register(function_table_entry());
   }
-}
+}  // namespace cuda
 
```

Comment:
nit: looks like accidental change given the same comment is on line 1187

Reply:
1174 should be removed